pom.xml

```diff
@@ -420,7 +421,7 @@
                         <excludes>
                             <exclude>**/*TestBase*.class</exclude>
                         </excludes>
-                        <argLine>-Xmx1G -Dlog4j.configuration=${log4j.properties}</argLine>
+                        <argLine>@{argLine} -Xmx1G -Dlog4j.configuration=${log4j.properties}</argLine>
```

Comment:
why is this required? seems like self-reflection.

Reply:
The late property evaluation is necessary in order to be able to provide a pointer to the jacoco runtime to the VM when testing. It seems that the @{argLine} property is set by jacoco and therefore different from the surefire parameter. the following two doc pages (not too long)  can probably explain that better than me:
* [jacoco:prepare-agent](https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html)
* [surefire](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html)

hth

Reply:
Guess if another plugin or profile modify argLine parameter `@{argLine}` makes sure to receive unmodified arguments. K its fine by me.